test/lock.cc

```diff
@@ -176,6 +179,8 @@ TEST(LockTest, Wait) {
 
   Interrupt interrupt;
 
+  // TODO(benh, xander): document why this is needed. Does anything assert that
+  // an Interrupt is registered? (For that matter, what's an Interrupt?)
```

Comment:
Moot point, since this test is deleted.

(I think that this test maybe just incorrectly copied `t1.Register()` from another test in this file that *actually* tested interrupt handling, but again, moot point!)